packages/byods/test/unit/spec/data-source-client/index.ts

```
@@ -0,0 +1,348 @@
+          jwsToken: 'someJwsToken',
+          createdBy: 'someUser',
+          createdAt: '2024-01-01T00:00:00Z',
+        },
+      };
+      httpClient.get.mockResolvedValue(response);
+      (decodeJwt as jest.Mock).mockReturnValue({
+        'com.cisco.datasource.url': 'https://mydatasource.com/handle',
+        sub: 'mySubject',
+        aud: 'myAudience',
+      });
+      jest.spyOn(global, 'setInterval').mockImplementation((fn: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)